<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>poky - Poky Build Tool and Metadata</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.yoctoproject.org/cgit/cgit.cgi/poky/atom/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/cgit.cgi/'><img src='/cgit/yocto-project-transp.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/cgit.cgi/'>index</a> : <a title='poky' href='/cgit/cgit.cgi/poky/'>poky</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='1.1_M1'>1.1_M1</option>
<option value='1.1_M2'>1.1_M2</option>
<option value='1.1_M3'>1.1_M3</option>
<option value='1.1_M4'>1.1_M4</option>
<option value='1.2_M1'>1.2_M1</option>
<option value='1.2_M2'>1.2_M2</option>
<option value='1.2_M3'>1.2_M3</option>
<option value='1.2_M4'>1.2_M4</option>
<option value='1.3_M1'>1.3_M1</option>
<option value='1.3_M2'>1.3_M2</option>
<option value='1.3_M3'>1.3_M3</option>
<option value='1.3_M4'>1.3_M4</option>
<option value='1.3_M5'>1.3_M5</option>
<option value='1.3_beta'>1.3_beta</option>
<option value='1.4_M1'>1.4_M1</option>
<option value='1.4_M3'>1.4_M3</option>
<option value='1.4_M4'>1.4_M4</option>
<option value='1.4_M5'>1.4_M5</option>
<option value='1.6_M5'>1.6_M5</option>
<option value='bernard'>bernard</option>
<option value='blinky'>blinky</option>
<option value='clyde'>clyde</option>
<option value='daisy'>daisy</option>
<option value='daisy-next'>daisy-next</option>
<option value='danny'>danny</option>
<option value='danny-next'>danny-next</option>
<option value='denzil'>denzil</option>
<option value='dora'>dora</option>
<option value='dora-toaster'>dora-toaster</option>
<option value='dylan'>dylan</option>
<option value='edison'>edison</option>
<option value='elroy'>elroy</option>
<option value='green'>green</option>
<option value='laverne'>laverne</option>
<option value='master' selected='selected'>master</option>
<option value='master-next'>master-next</option>
<option value='pinky'>pinky</option>
<option value='purple'>purple</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>Poky Build Tool and Metadata</td><td class='sub right'>git repository hosting</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/cgit.cgi/poky/about/'>about</a><a href='/cgit/cgit.cgi/poky/'>summary</a><a href='/cgit/cgit.cgi/poky/refs/'>refs</a><a href='/cgit/cgit.cgi/poky/log/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>log</a><a class='active' href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>tree</a><a href='/cgit/cgit.cgi/poky/commit/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>commit</a><a href='/cgit/cgit.cgi/poky/diff/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>diff</a><a href='/cgit/cgit.cgi/poky/stats/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/cgit.cgi/poky/log/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/cgit.cgi/poky/tree/'>root</a>/<a href='/cgit/cgit.cgi/poky/tree/meta'>meta</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core'>recipes-core</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus'>dbus</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2'>dbus-glib-0.100.2</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>test-install-makefile.patch</a></div><div class='content'>blob: 2e8e178bb7e7e25482021da6bca56b49c05acfef (<a href='/cgit/cgit.cgi/poky/plain/meta/recipes-core/dbus/dbus-glib-0.100.2/test-install-makefile.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
<a id='n44' href='#n44'>44</a>
<a id='n45' href='#n45'>45</a>
<a id='n46' href='#n46'>46</a>
<a id='n47' href='#n47'>47</a>
<a id='n48' href='#n48'>48</a>
</pre></td>
<td class='lines'><pre><code>Change Makefile.am to install regression tests for test package purpose.

Upstream-Status: Inappropriate [test not install is for purpose from upstream]

Signed-off-by: Yao Zhao &lt;yao.zhao@windriver.com&gt;

diff --git a/test/Makefile.am b/test/Makefile.am
index 7ba11a8..249f0e7 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -13,7 +13,8 @@ else
 TEST_BINARIES=
 endif
 
-noinst_PROGRAMS= $(TEST_BINARIES)
+testdir = $(datadir)/@PACKAGE@/tests
+test_PROGRAMS= $(TEST_BINARIES)
 
 test_service_SOURCES=				\
 	test-service.c
diff --git a/test/interfaces/Makefile.am b/test/interfaces/Makefile.am
index 3cb2c39..e6de67d 100644
--- a/test/interfaces/Makefile.am
+++ b/test/interfaces/Makefile.am
@@ -39,7 +39,8 @@ if DBUS_BUILD_TESTS
 
 ## we use noinst_PROGRAMS not check_PROGRAMS for TESTS so that we
 ## build even when not doing "make check"
-noinst_PROGRAMS = test-service test-client
+testdir = $(datadir)/@PACKAGE@/tests/interfaces
+test_PROGRAMS = test-service test-client
 
 test_service_SOURCES = \
 	test-interfaces.c \
diff --git a/test/core/Makefile.am b/test/core/Makefile.am
index ef6cb26..9786c3a 100644
--- a/test/core/Makefile.am
+++ b/test/core/Makefile.am
@@ -46,7 +46,8 @@ endif
 
 ## we use noinst_PROGRAMS not check_PROGRAMS for TESTS so that we
 ## build even when not doing "make check"
-noinst_PROGRAMS = \
+testdir = $(datadir)/@PACKAGE@/tests/core
+test_PROGRAMS = \
 	test-dbus-glib \
 	test-service-glib \
 	$(THREAD_APPS) \
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-07-21 11:20:30 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
